desktop/libs/indexer/src/indexer/api3.py

```diff
@@ -153,6 +161,14 @@ def guess_format(request):
       }
     })
     _convert_format(format_)
+    if file_format["path"][-3:] == 'xls' or file_format["path"][-4:] == 'xlsx': 
+      format_ = {
+          "quoteChar": "\"",
+          "recordSeparator": '\\n',
+          "type": "excel",
+          "hasHeader": True,
+          "fieldSeparator": ","
+        }
```

Comment:
Nit: space before and after this code block.

Reply:
done